orchestrator/src/core/client/alert/error.rs

```diff
@@ -8,6 +8,9 @@ pub enum AlertError {
     #[error("Topic ARN is empty")]
     TopicARNEmpty,
 
+    #[error("Topic ARN is invalid")]
+    TopicARNInvalid,
```

Comment:
```suggestion
    #[error("Topic ARN is invalid, ARN : {0}")]
    TopicARNInvalid(String),
```

since it is invalid ARN can you parse the invalid string here as part of the error message and argument

Reply:
Seconding this!

Reply:
Valid, added.